src/PublishPort.jsx

```diff
@@ -57,8 +107,14 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                             step={1}
                             min={1}
                             max={65535}
+                            validated={validationFailed?.containerPort ? "error" : "default"}
                             value={item.containerPort || ''}
-                            onChange={(_event, value) => onChange(idx, 'containerPort', value)} />
+                            onChange={(_event, value) => {
+                                utils.validationClear(validationFailed, "containerPort", onValidationChange);
+                                utils.validationDebounce(() => onValidationChange(validatePublishPort(value, "containerPort", validationFailed)));
+                                onChange(idx, 'containerPort', value);
+                            }} />
+                <FormHelper helperTextInvalid={validationFailed?.containerPort} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231009-115441-607719f9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)